src/test/article.spec.js

```diff
@@ -0,0 +1,132 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import BaseRepository from '../repository/base.repository';
+import { createUser, createArticle, getArticle } from './utils/helpers';
+import db from '../database/models';
+import helper from '../helpers/utils';
+
+const ARTICLES_API = '/api/v1/articles';
+
+chai.use(chaiHttp);
+
+const server = () => chai.request(app);
+
+describe('PATCH api/v1/articles/bookmark', () => {
+  beforeEach(async () => {
+    await db.Bookmark.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should bookmark an article', async () => {
+    const firstUser = await createUser();
+    const secondUser = await createUser();
+    const theArticle = await createArticle(
+      await getArticle({ authorId: secondUser.id })
+    );
+    const numberOfArticles = await BaseRepository.findAll(db.Article);
+    const numberOfBookmarks = await BaseRepository.findAll(db.Bookmark, {
```

Comment:
You can use sequelize `count`. You can rename to `bookmarksCount`

Reply:
Change implemented thank you